server.js

```diff
@@ -98,8 +98,8 @@ var config = configure()
 
 var server = require('webserver').create();
 var urlPrefix = config['url'];
-var port = config['port'] || system.env.PORT || 8082;
-var key = config['key'];
+var port = system.env.PORT || config['port'] || 8082;
+var key =  system.env.KEY || config['key'];
```

Comment:
Eehm. I guess so, we have

1. Hardcoded
2. config.json
3. /etc/prerender.json
4. config-local.json
5. environment

So I guess for current Heroku setup:
1. We could just "fail" if no value was found,
2. should be left there I guess, it is handy for having a place to put settings
3. could be cut out for heroku, but handy for deployment on non-heroku setups 
4. is really handy for local development, 
5. is needed for Heroku...